.github/workflows/main.yml

```
@@ -19,5 +18,14 @@ jobs:
           node-version: '20'
       - name: Install dependencies
         run: npm install
+      - name: Install Husky hooks
+        run: npx husky install
+      - name: Cache node_modules
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)